.bumpedrc

```diff
@@ -0,0 +1,36 @@
+files: [
+  'package.json'
+]
+
+plugins:
+
+  prerelease:
+
+    'Linting config files':
+      plugin: 'bumped-finepack'
+
+    'Linting projects files':
+      plugin: 'bumped-terminal'
+      command: 'npm run lint'
+
+    'Preparing build':
+      plugin: 'bumped-terminal'
+      command: 'npm run prepublish'
+
+  postrelease:
+
+    'Generating CHANGELOG file':
+      plugin: 'bumped-changelog'
+
+    'Committing the new version':
+      plugin: 'bumped-terminal'
+      command: 'git add . && git commit -m "$newVersion releases"'
+
+    'Publishing at Github':
+      plugin: 'bumped-terminal'
+      command: 'git tag $newVersion && git push --tags && git push origin master'
```

Comment:
We cannot get rid of this command. `Master` branch should contain code from tags, especially changelog. Additionally I don't like `git add .` at all. It may add unexpected files to our repo. We always should explicitly choose files.
In my opinion something like this would be better:
```
    'Generating CHANGELOG file':
      plugin: 'bumped-changelog'

    'Publishing CHANGELOG at Github':
      plugin: 'bumped-terminal'
      command: 'git add CHANGELOG.md && git commit CHANGELOG.md -m "$newVersion release" && git push origin master'

    'Publishing tag at Github':
      plugin: 'bumped-terminal'
      command: 'git tag $newVersion && git push origin $newVersion'

    'Publishing at NPM':
      plugin: 'bumped-terminal'
      command: 'npm publish'
```